openedx/features/partners/helpers.py

```diff
@@ -107,3 +109,13 @@ def get_partner_from_user(user):
 
 def user_has_performance_access(user, partner):
     return bool(partner and user.has_perm('partners.%s' % PERFORMANCE_PERM_FRMT.format(slug=partner.slug)))
+
+
+def get_partner_registration_countries(partner_slug):
+    sorted_countries = sorted(COUNTRIES.items(), key=operator.itemgetter(1))
+    top_registration_countries = PARTNERS_TOP_REGISTRATION_COUNTRIES.get(partner_slug, None)
+    if top_registration_countries:
+        top_countries = [(country[0], COUNTRIES.get(country[0], None)) for country in top_registration_countries]
```

Comment:
apparently you are doing unwanted loop here.

Reply:
You are right, but maybe there can be a case where we want to get the country tuples from our global ones, instead of just serving the ones we saved in constants?